### PR TITLE
Add tests for JsonAdaptedConsultation

### LINKED ISSUE #175: Add Consultation Test at Storage
Insert JsonAdaptedConsultation tests at test/storage/.

### LINKED ISSUE #176: Write tests for JsonAdaptedConsultation

### PR DESCRIPTION
**Pull request by chrisgzf**
This PR's base is Rishi's PR #172, ~somehow GitHub won't let me set his fork as base.~

~EDIT: Oh LMAO I realised why... I think for me to be able to set his fork as the base, I need to PR to his fork instead... #forkingworkflowsucks Nvm, let's wait for #172 to be merged, then this PR will sync up with master correctly. **(Until that happens, don't review yet, cos the diff will be harder to read)**~

This PR adds tests for JsonAdaptedConsultation.

Closes #176, #175

### CODE REVIEW

File: src/test/java/seedu/address/storage/JsonAdaptedConsultationTest.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    private static final Consultation BENSON_CONSULT = BENSON_GROUP_CONSULT;
+    private static final String VALID_NAME = BENSON.getName().toString();
+    private static final String VALID_DAY = TypicalConsults.BENSON_DAY;
+    private static final String VALID_TIME = TypicalConsults.BENSON_TIME;
+    private static final String VALID_LOCATION = TypicalConsults.BENSON_LOCATION;
+    private static final String VALID_CONSULT_TYPE = TypicalConsults.BENSON_CONSULT_TYPE;
+
+
+    @Test
+    public void toModelType_validPersonDetails_returnsPerson() throws Exception {
```
Comment (Rishi5154) [inline src/test/java/seedu/address/storage/JsonAdaptedConsultationTest.java]: Rename the tests containing person to consultations/consults

Comment (chrisgzf) [reply src/test/java/seedu/address/storage/JsonAdaptedConsultationTest.java]: Nice catch! Thanks!

Comment (chrisgzf) [reply src/test/java/seedu/address/storage/JsonAdaptedConsultationTest.java]: Fixed in https://github.com/AY2021S1-CS2103T-F11-1/tp/pull/182/commits/37685c47c5ac4632eb43c995b8a213c4d0867dfc

**Review by kwannoel — APPROVED** [review]
LGTM.

**Pull request merged**